tests/vm_test.go

```diff
@@ -166,7 +166,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(createdVM.Spec.Template.Spec.Domain.Machine.Type).To(Equal(testingMachineType))
 		})
 
-		It("should keep the supplied MachineType when created", func() {
+		It("[test_id:3312]should keep the supplied MachineType when created", func() {
```

Comment:
The test_id here and at line 157 are the same. Tried looking at polarion, could not find at a quick glance. May need to revisit this.

Reply:
Fixed